spec/features/irs_attempts_api/event_tracking_spec.rb

```diff
@@ -30,12 +30,18 @@
     sign_in_user(user)
 
     events = irs_attempts_api_tracked_events
-    expected_event_types = ['email-and-password-auth', 'mfa-phone-verification-otp-sent']
+    expected_event_types = %w[email-and-password-auth mfa-phone-verification-otp-sent]
 
     received_event_types = events.map(&:event_type)
 
-    expect(events.count).to be > 0
-    expect(received_event_types.sort).to eq(expected_event_types.sort)
+    expect(events.count).to eq received_event_types.count
+    expect(received_event_types).to match_array(expected_event_types)
+
+    metadata = events.first.event_metadata
+    expect(metadata[:user_ip_address]).to eq '127.0.0.1'
+    expect(metadata[:irs_application_url]).to eq 'http://localhost:7654/auth/result'
+    expect(metadata[:unique_session_id]).to be_a(String)
+    expect(metadata[:success]).to be_truthy
```

Comment:
I'm kind of ad-libbing stuff here. I wanted to test some attributes of the events, but didn't want to go through a ton of work in this file to mock out all the request parameters. Is this an improvement, or is doing some but not all of this worse than nothing?